model/ResultsService.php

```diff
@@ -1312,14 +1317,24 @@ private function convertDates(&$data)
      *
      * @return bool
      */
-    private function filterData($row, array $filters)
+    private function filterData($row, array $filters, string $deliveryExecutionIdentifier)
```

Comment:
This is not backward compatible. Have you checked all related usages that will be impacted?

Reply:
Yep